Containerfile

```diff
@@ -0,0 +1,32 @@
+# Use the NODE_VERSION argument to specify the Node.js version
+ARG NODE_VERSION
+
+# Use the specified Node.js version as the base image
+FROM node:${NODE_VERSION}-alpine
+
+# Set Node args
+ENV NODE_OPTIONS=--openssl-legacy-provider

```

Comment:
This line seems to be the reason I get errors when building it with node 16. Starting with node 18 this doesn't seem to bother the builder so I'm fine with that.

Error for reference:
```
 > [6/6] RUN npm ci:
#11 0.182 node: --openssl-legacy-provider is not allowed in NODE_OPTIONS
```

Reply:
Yes I forgot to mention that setting should be ommited in version 16, my bad.

Reply:
Looks good!